packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/FloatFixedEnumWithIntValue.cs

```diff
@@ -0,0 +1,13 @@
+// <auto-generated/>
+
+#nullable disable
+
+namespace UnbrandedTypeSpec.Models
+{
+    public enum FloatFixedEnumWithIntValue
+    {
+        One = 1,
```

Comment:
well in fixed enum, typespec does not have a way to let us specify this is a float value (even if we put 1.0).
But in the fixed enum, it will never be a breaking change in the future if some one add a new float value such as `3.14`

Reply:
can you elaborate on this?  Are you saying if you set a fixed enum to `1.0` it will come to us as `1`?

Reply:
this might be an issue on upstream. I will investigate and fix it in other PR.

Reply:
opened an issue in TCGC: https://github.com/Azure/typespec-azure/issues/932